### PR TITLE
fix overflow by int32 mul test=develop

### PR DESCRIPTION
**Pull request by Aurelius84**
修复int32相乘overflow的问题

### CODE REVIEW

**Comment by luotao1** [comment]
Each commit should add `test=develop` to trigger the CI.

File: paddle/fluid/operators/sequence_ops/sequence_conv_op.h [hunk 1]
```diff
@@ -41,15 +41,15 @@ class SequenceConvKernel : public framework::OpKernel<T> {
 
     PADDLE_ENFORCE_EQ(in->lod().size(), 1UL,
                       "Only support one level sequence now.");
-
-    const Tensor* padding_data = nullptr;
```
Comment (liupluswei) [inline paddle/fluid/operators/sequence_ops/sequence_conv_op.h]: I think modify the [ContextProjectFunctor](https://github.com/PaddlePaddle/Paddle/blob/6864370a9e7ba991b0dba2553150901bcf63234f/paddle/fluid/operators/math/context_project.h#L90) API will be cleaner, update the  **const Tensor& padding_data** parameter to  **const Tensor* padding_data**, and inside the API, if **padding_trainable** is true, PADDLE_ENFORCE_NOT_NULL(padding_data)

Comment (Aurelius84) [reply paddle/fluid/operators/sequence_ops/sequence_conv_op.h]: Thanks，API has been modified from reference to point.

**Review by liupluswei — APPROVED** [review]
LGTM

**Pull request merged**